instrumentation/opentelemetry-instrumentation-wsgi/setup.py

```diff
@@ -11,6 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+# DO NOT EDIT. THIS FILE WAS AUTOGENERATED FROM templaes/wsgi.
+# Run `python scripts/generate_setuppy.py` to regenerate.
```

Comment:
Nit: I see why you named this the way you did, but I would rename the script `generate_setup.py`, it's easier to read.